optics/src/main/scala/io/circe/optics/JsonPath.scala

```diff
@@ -1,10 +1,12 @@
 package io.circe.optics
 
-import io.circe.{ Decoder, Encoder, Json }
-import io.circe.optics.JsonOptics._, io.circe.optics.JsonObjectOptics._
-import monocle.{ Optional, Prism }
-import monocle.function.index
+import io.circe.optics.JsonObjectOptics._
+import io.circe.optics.JsonOptics._
+import io.circe.{Decoder, Encoder, Json}
```

Comment:
This is super nitpicky, but do you mind maintaining the alphabetization (where capital letters precede lowercase) and putting a space inside the `{ }`? (I could also do this in a commit to the `topic/optics` branch after merging).